draft-ietf-rats-corim.md

```diff
@@ -1633,9 +1618,9 @@ ae = [
 ]
 ~~~
 
-| Type | `e` | `c` | `a` | `ns` | `cm` | `p` |
+| Type | `en` | `el` | `a` | `cm` | `p` |
```

Comment:
The shorthand convention doesn't seem to be defined anywhere.  OK, it's not completely unintuitive :-) but can't we use the unabbreviated labels?